src/pages/ProductDetail/Overview/SelectOption/SelectedProduct/SelectedProduct.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+import './SelectedProduct.scss';
+
+class SelectedProduct extends Component {
+  getSelectedProduct = (e, targetProductInfo) => {
+    this.props.takeCountValue(targetProductInfo, e.target.value);
+  };
+
+  handleDeleteButton = (productIndex) => {
+    this.props.takeDeleteFuction(productIndex);
+  };
+
+  render() {
+    const counts = [1, 2, 3, 4, 5, 6, 7, 8, 9];
```

Comment:
```suggestion
Array.from({length: 9}, (_, i) => i + 1);
```

변하지 않는 상수변수라면 컴포넌트 위에 const COUNTS로 빼주세요~

Reply:
컴플리뜨!